packages/angular/src/generators/setup-mf/lib/add-cypress-workaround.ts

```diff
@@ -3,13 +3,12 @@
 // is attached to the index.html with type=module
 
 import type { ProjectConfiguration, Tree } from '@nrwl/devkit';
-import type { Schema } from '../schema';
-
 import {
   joinPathFragments,
-  readProjectConfiguration,
   logger,
+  readProjectConfiguration,
 } from '@nrwl/devkit';
+import type { Schema } from '../schema';
 
 export function addCypressOnErrorWorkaround(tree: Tree, schema: Schema) {
   const e2eProjectName = schema.e2eProjectName ?? `${schema.appName}-e2e`;
```

Comment:
I think we should do nothing here if the `e2eTestRunner` option is set to `none`. Otherwise, we'd get the warning even if we choose to not generate any e2e tests.

Reply:
good idea